### PR TITLE
Renaming measure to metric

### PR DESCRIPTION
**Pull request by dlcd2203**
JIRA: SD-1921

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [x] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6093/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7875/ : FAILURE in 4m 08s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6174/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7954/ : SUCCESS in 16m 41s

**Comment by dlcd2203** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1267/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6175/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1267/ : UNSTABLE in 19m 36s

**Comment by dlcd2203** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6176/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7955/ : SUCCESS in 18m 09s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1268/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6177/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1268/ : UNSTABLE in 21m 47s

File: libs/sdk-ui-ext/src/internal/translations/en-US.json [hunk 1]
```diff
@@ -294,27 +399,53 @@
...
+        "comment": "Do not translate HTML markup enclosed in <>.  Do not translate {icons}. Translate words 'stack by'.",
+        "limit": 0
+    },
     "dashboard.bucket.metric_view_by_warning": {
         "value": "To add additional measure, remove {icons} from '<span class=\"stack-by\">'view by'</span>'",
         "comment": "Do not translate HTML markup enclosed in <>. Do not translate {icons}. Translate words 'view by'.",
         "limit": 0
     },
+    "dashboard.bucket.metric_view_by_warning|metric": {
+        "value": "To add additional measure, remove {icons} from <span class=\"stack-by\">view by</span>",
```
Comment (anvuong119) [inline libs/sdk-ui-ext/src/internal/translations/en-US.json:423]: `measure` here should be changed to `metric`, shouldn't it?

File: libs/sdk-ui-ext/src/internal/translations/en-US.json [hunk 2]
```diff
@@ -914,16 +1132,31 @@
...
         "limit": 0
     },
+    "dashboard.bucket.tertiary_measures_title.bullet|metric": {
+        "value": "Metric",
+        "comment": "",
+        "limit": 0
+    },
     "dashboard.bucket.measures_subtitle.bullet": {
         "value": "Primary",
         "comment": "Primary measure of a bullet chart.",
```
Comment (anvuong119) [inline libs/sdk-ui-ext/src/internal/translations/en-US.json:1162]: There are texts with `measure` in [libs/sdk-ui/src/base/localization/bundles/en-US.json](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui/src/base/localization/bundles/en-US.json#L63) and  [libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json) as well, for example:
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui/src/base/localization/bundles/en-US.json#L63
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui/src/base/localization/bundles/en-US.json#L68
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json#L637
- https://github.com/gooddata/gooddata-ui-sdk/blob/7dcb91093ee2ce8a4f7205674160118d75d78d25/libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json#L642

→ Please don't forget to add new texts containing `metric` in those 2 files.

File: libs/sdk-backend-spi/src/common/settings.ts [hunk 3]
```diff
@@ -187,6 +187,11 @@ export interface ISettings {
     enableAnalyticalDashboardPermissions?: boolean;
 
     [key: string]: number | boolean | string | object | undefined;
+
+    /**
+     * Enable renaming measure to metric in AD&KD
```
[the remark this answers is not in the document]
Comment (dlcd2203) [reply libs/sdk-backend-spi/src/common/settings.ts]: fixed

File: libs/sdk-ui-ext/src/internal/translations/en-US.json (hunk 1 above)
Comment (dlcd2203) [reply libs/sdk-ui-ext/src/internal/translations/en-US.json]: fixed

File: libs/sdk-ui-ext/src/internal/translations/en-US.json (hunk 2 above)
Comment (dlcd2203) [reply libs/sdk-ui-ext/src/internal/translations/en-US.json:1162]: fixed

**Comment by dlcd2203** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6178/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7956/ : SUCCESS in 19m 09s

**Comment by dlcd2203** [comment]
extended check cypress

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1269/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6179/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1269/ : UNSTABLE in 24m 45s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**drilling.spec.ts**|2|-|9.7700s|
|✅|**dateFilter.spec.ts**|7|-|13.4600s|
|✅|**dashboard.spec.ts**|9|-|14.2010s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/91/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6180/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/91/ : SUCCESS in 8m 40s

**Comment by dlcd2203** [comment]
extended check sonar

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 4]
```diff
@@ -64,12 +64,23 @@
...
+    },
     "visualizations.totals.dropdown.tooltip.nat.disabled.mvf|report": {
         "value": "Rollup (Total) aggregation is not available when filtering the report by measure value. To use Rollup (Total), remove all measure value filters from the filter bar.",
         "comment": "The tooltip that is shown in pivot's table totals menu when rollup (native) total is disabled because measure value filters are configured for the pivot.",
         "limit": 0,
         "translate": false
     },
+    "visualizations.totals.dropdown.tooltip.nat.disabled.mvf|report|metric": {
+        "value": "Rollup (Total) aggregation is not available when filtering the report by metric value. To use Rollup (Total), remove all metric value filters from the filter bar.",
+        "comment": "The tooltip that is shown in pivot's table totals menu when rollup (native) total is disabled because measure value filters are configured for the pivot.",
```
Comment (anvuong119) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:80]: `measure` in the `"comment"` should be changed to `metric` as well, shouldn't it?

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR2011

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6182/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/856/ : SUCCESS in 17m 14s

File: libs/sdk-ui/src/base/localization/bundles/en-US.json (hunk 4 above)
Comment (dlcd2203) [reply libs/sdk-ui/src/base/localization/bundles/en-US.json]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6183/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7959/ : SUCCESS in 16m 26s

**Comment by anvuong119** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/6189/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7963/ : SUCCESS in 19m 49s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2011

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1776/ : SUCCESS in 1m 53s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2057/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/857/ : SUCCESS in 13m 29s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7964/ : SUCCESS in 16m 38s

**Pull request merged**